### ISSUE TITLE
Video Tutorial / Showcase

### ISSUE THREAD

**Issue by daisyUniverse** [robinuniverse/Keebie#36]
Alright, so here's where I'll post updates on the progress of the video.

1. install it from scratch, preferably on several systems to get a feel for the process
2. begin writing the script ( and possibly update the readme while I'm at it )
3. record the process of installation, and record some VO
4. mix it all together in kdenlive

Some useful things to do might also be to reread the entire codebase in order to familiarize myself with any features we've added in but haven't documented well

**Comment by Michael-E-B** [comment]
> [...] reread the entire codebase in order to familiarize myself with any features we've added in but haven't documented well

Yeah... I'll be honest there at least a bit of mess in there. Layer variables are almost entirely unacknowledged (and must be added to layers manually with a text editor), the full capability of Keebie's tapdance system isn't really well explained (a shame, it's my favorite feature), the LEDs feature could do with some touching up, and I'm probably forgetting a few things.
